tests/test_adjoint.py

```diff
@@ -26,13 +26,10 @@ def test_gradcheck() -> None:
     }
     state = random_state(MAX_QUBITS)
 
-    def adjoint_expfn(values) -> Array:
-        return adjoint_expectation(state, ops, observable, values)[0]
+    def exp_fn(values: dict, diff_mode: DiffMode) -> Array:
```

Comment:
Shouldn't `diff_mode` be defaulted ?

Reply:
I think the `expectation` function has a default. For the test, it's fine to keep it like this as we compare the `adjoint` with the `ad` diff

Reply:
Ok we fixed it in the test.